freeipa/src/main/resources/init/init.ftl

```diff
@@ -15,6 +15,19 @@ export SSH_USER=${sshUser}
 export SALT_BOOT_PASSWORD=${saltBootPassword}
 export SALT_BOOT_SIGN_KEY=${signaturePublicKey}
 export CB_CERT=${cbCert}
+<#if proxyEnabled!false>
+export IS_PROXY_ENABLED=true
+export PROXY_HOST=${proxyHost}
+export PROXY_PORT=${proxyPort}
+<#if proxyUser??>
+export PROXY_USER="${proxyUser}"
+</#if>
+<#if proxyPassword??>
+export PROXY_PASSWORD="${proxyPassword}"
```

Comment:
I don't know if it's a valid case where we have password and no user, so maybe it would make sense to move the password if into the user if